RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -40,6 +40,9 @@ extension Package: VariableDataProvider {
     }
 
     func periodName(_ locale: Locale) -> String {
+        if packageType == .custom {
```

Comment:
The issue seems to be that `PackageType.localizationKey` returns `nil` for `.custom` **but also `.unknown`**. 
I'd suggest a slightly different implementation to fix `.unknown` too: change `Localization.localized(packageType:locale:)` to return an `Optional`, and provide `self.identifier` as the fallback here.